### PR TITLE
Add: hadialqattan's go implementation

### PR DESCRIPTION
**Pull request by hadialqattan**
LC-3 VM written in Go.

### CODE REVIEW

**Comment by justinmeiners** [comment]
Thanks for the contribution. I hope you enjoyed it.

We do already have 2 Go implementations.
Is there anything unique or interesting about yours that would not be in those other two?

**Comment by hadialqattan** [comment]
My pleasure, yeah I enjoyed it.

I see your point, My implementation is different + I've created a very simple method (based on Docker) to run any pure LC-3 assembly program using a simple command, if someone comes to run an assembly program, he can just put it in the `programs/source/` directory and then run this simple command `./compiler/runner.sh program_name`.
[the compiled file will be saved on `programs/bin/`]

In short words, my project includes an assembly compiler & runner (based on Docker, script).


Thanks a lot for the horrible article, I learned a lot from it!

**Comment by justinmeiners** [comment]
> Thanks a lot for the horrible article, 

lol.

Cool, thanks for the explanation. I will get that added.

**Comment by hadialqattan** [comment]
> > Thanks a lot for the horrible article,
> 
> lol.
> 
> Cool, thanks for the explanation. I will get that added.

Thanks again, have a good day!
